### PR TITLE
SCRD-5570 Move password fields to forms

### PR DESCRIPTION
**Pull request by eapodaca**
* Move password fields to forms
* Refactor touched classes away from class property syntax
* Use function bind syntax when needed

### CODE REVIEW

File: src/components/Modals.js [hunk 1]
```diff
@@ -110,14 +109,14 @@ class GetSshPassphraseModal extends Component {
...
-  isValidPassphrase = () => {
+  isValidPassphrase() {
     if (this.state.passphrase) {
       return true;
     }
     return false;
   }
 
-  renderErrorMessage = () => {
+  renderErrorMessag() {
```
Comment (GarySmith) [inline src/components/Modals.js]: This incorrectly renamed the function (missing trailing 'e')

Comment (eapodaca) [reply src/components/Modals.js]: Good catch.

**Comment by GarySmith** [comment]
With this code in place, I am seeing errors in the Javascript console on at least one screen:
Use the ?start=installer query parameter to go into the installer, and go next until you get the "Assign Servers and Assign Server Roles" page, then click on the edit icon on one of the servers on the right-hand side of the page. When that dialog is drawn, you get errors about missing "key" props and about missing 'onAction' event handler.

**Comment by eapodaca** [comment]
I've resolved the issue the `onAction` error. having trouble understanding how any of these changes would cause the `key` error.

**Comment by GarySmith** [comment]
https://reactjs.org/docs/lists-and-keys.html may help if you haven't already seen it.

**Comment by GarySmith** [comment]
I don't see the warnings any more

**Comment by jeremy-moffitt** [comment]
trying to test this but I'm just getting the "Loading... Please Wait" whether its in Day 0 or Day 2, with no errors in the console... might be an env issue, still debugging

**Comment by jeremy-moffitt** [comment]
switched to master and seeing the same thing, so I must have an env issue... will debug that and try again

**Pull request merged**